File: tea-server/src/Tea.Domain/Entities/Size.cs
namespace Tea.Domain.Entities
{
    public class Size
    {
        public int Id { get; set; }
        public required string Name { get; set; }
        public string? Description { get; set; }
        public bool IsDeleted { get; set; }
        public decimal Price { get; set; }
        public decimal NewPrice { get; set; }
        public int ItemId { get; set; }
        public Item? Item { get; set; }
    }
}

File: tea-manage/src/app/shared/models/size.ts
export interface Size {
  id: number;
  name: string;
  description: string;
  price: number;
  newPrice: number;
}

export interface SizeAdd {
  name: string;
  description: string;
  price: number;
  newPrice: number;
}

export interface SizeUpdate {
  id: number;
  name: string;
  description: string;
  price: number;
  newPrice: number;
  itemId: number;
}
